src/components/Global/Input/Input.module.scss

```diff
@@ -0,0 +1,13 @@
+@import 'src/styles/abstracts/_mixins/text';
+@import 'src/styles/abstracts/_variables';
+
+.input {
+  @include font-button;
+  color: $black;
+  font-size: 0.875rem;
+  width: 100%;
+  padding: 0.5rem 1rem;
+  border: 2px solid #4b4b4b;
+  box-sizing: border-box;
+  border-radius: 999px;
```

Comment:
still need to change to rem

Reply:
This one is actually okay, since it should be maximum roundness regardless of font size